neuro_extras/main.py

```diff
@@ -217,6 +222,54 @@ def image_copy(source: str, destination: str) -> None:
     run_async(_copy_image(source, destination))
 
 
+@storage.command("copy")
+@click.argument("source")
+@click.argument("destination")
+def cluster_copy(source: str, destination: str) -> None:
+    run_async(_copy_storage(source, destination))
+
+
+async def _copy_storage(source: str, destination: str) -> None:
+    src_uri = URL(source)
+    src_cluster = src_uri.host
+    src_path = src_uri.parts[2:]
+
+    dst_uri = URL(destination)
+    dst_cluster = dst_uri.host
+    dst_path = dst_uri.parts[2:]
+
+    assert src_cluster
+    assert dst_cluster
+    async with neuro_api.get() as client:
+        await client.config.switch_cluster(dst_cluster)
+        await client.storage.mkdir(URL("storage:"), parents=True, exist_ok=True)
+    await _run_copy_container(src_cluster, "/".join(src_path), "/".join(dst_path))
+
+
+async def _run_copy_container(
+    src_cluster: str, src_path: str, dst_path: str
+) -> Process:
+    args = [
+        "neuro",
+        "run",
+        "-s",
+        "cpu-small",
+        "--pass-config",
+        "-v",
+        "storage:://storage",
+        "-e",
+        f"NEURO_CLUSTER={src_cluster}",
+        "neuromation/neuro-extras:latest",
+        f'"cp -r -u -T storage:{src_path} /storage/{dst_path}"',
```

Comment:
I know, this command is already well-tested and well-used, but wouldn'it it better to use `rclone`? (so that we'd need to install it in `neuromation/neuro-extras:latest` as well).

Reply:
or at least to have a progress and some kind of confirmation at the end? for example:
`neuro run ... bash -c 'cp -r --progress... && echo Data copied from {SRC} to {DST}'`

Reply:
Entrypoint in the container runs `neuro $1`, so adding an echo at the end is solveable, but a bit problematic. The `--progress` parameter has been added though.